.github/workflows/ci.yaml

```
@@ -32,6 +32,11 @@ jobs:
             - name: Build
               run: npm run build
 
+            - name: Add missing hostname to /etc/hosts
```

Since it is not possible to add a comment to a workflow step, maybe a comment like this could be an alternative to also have documentation of the rationale for this step within the workflow file?

```suggestion
            # Workaround for issue #1138
            - name: Add missing hostname to /etc/hosts
```